definitions/divorce/json/ComplexTypes/ComplexTypes-deemed-and-dispensed-nonprod.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "LiveFrom": "24/09/2020",
+    "ID": "DivorceServiceApplicationDocument",
+    "ListElementCode": "Document",
+    "FieldType": "DivorceDocument",
+    "ElementLabel": "Service Application",
+    "SecurityClassification": "Public"
+  }
+]
```

Comment:
Defining collection tpe

Reply:
if you need to store just `DivorceDocument` there is no need of defining your own complex type.

You can use:

```
{
    "LiveFrom": "24/09/2020",
    "CaseTypeID": "DIVORCE",
    "ID": "ServiceApplicationDocuments",
    "Label": "Service Application Documents",
    "FieldType": "Collection",
    "FieldTypeParameter": "DivorceDocument",
    "SecurityClassification": "Public"
  },
```

Reply:
@yarpo Updated